app/controllers/optIn/OptInCompletedController.scala

```diff
@@ -17,20 +17,23 @@
 package controllers.optIn
 
 import auth.{FrontendAuthorisedFunctions, MtdItUser}
-import config.{AgentItvcErrorHandler, FrontendAppConfig, ItvcErrorHandler}
 import config.featureswitch.FeatureSwitching
+import config.{AgentItvcErrorHandler, FrontendAppConfig, ItvcErrorHandler}
 import controllers.agent.predicates.ClientConfirmedController
+import models.optin.OptInCompletedViewModel
 import play.api.Logger
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
 import services.optIn.OptInService
 import utils.AuthenticatorPredicate
+import views.html.optIn.{OptInCompletedCurrentYearView, OptInCompletedNextYearView}
 
-import java.time.LocalDateTime
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class OptInCompletedController @Inject()(val optInService: OptInService,
+class OptInCompletedController @Inject()(val currentYearView: OptInCompletedCurrentYearView,
+                                         val nextYearView: OptInCompletedNextYearView,
```

Comment:
I'm unsure if having separate views for the current and next year will be the best way forward here. There's a number of further scenarios for this page where variations would cut across both of these. We're still working out what needs to be done and making the tickets. Let me catch up with the BA and try to get a clear idea of what needs to be done, and then let's talk this through and find the best way to deliver the design.